runtime/src/lib.rs

```diff
@@ -533,7 +533,7 @@ pub type Executive = frame_executive::Executive<
 	frame_system::ChainContext<Runtime>,
 	Runtime,
 	AllPalletsWithSystem,
-	(AddOnchainSettings<Runtime>, AddSealToLoc<Runtime>),
+	(AddOnchainSettings<Runtime>, AddLicenseToCollectionItem<Runtime>),
```

Comment:
You may also safely remove the `AddOnchainSettings` migration.

Reply:
Done.